gadcp/io.py

```diff
@@ -208,3 +206,64 @@ def _is_number(s):
         return True
     except ValueError:
         return False
+
+
+def cf_conventions():
+    """Return dictionary with CF standard names and units.
+
+    See https://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html
+    and http://cfconventions.org/cf-conventions/cf-conventions.html
+    for further information.
+
+    Returns
+    -------
+    CF : dict
+        CF standard names in units.
+    """
+
+    CF = dict(
+        depth=dict(
+            long_name="depth",
+            standard_name="depth",
+            units="m",
+            positive="down",
+        ),
+        u=dict(
+            long_name="u", standard_name="eastward_sea_water_velocity", units="m s-1"
+        ),
+        v=dict(
+            long_name="v", standard_name="northward_sea_water_velocity", units="m s-1"
+        ),
+        w=dict(long_name="w", standard_name="upward_sea_water_velocity", units="m s-1"),
+        e=dict(
+            long_name="error velocity",
+            standard_name="indicative_error_from_multibeam_acoustic_doppler_velocity_profiler_in_sea_water",
+            units="m s-1",
+        ),
+        pressure=dict(
+            long_name="pressure", standard_name="sea_water_pressure", units="dbar"
+        ),
+        temperature=dict(
+            long_name="temperature",
+            standard_name="sea_water_temperature",
+            units="degrees_Celsius",
+        ),
+        xducer_depth=dict(
+            long_name="transducer depth",
+            standard_name="transducer_depth",
```

Comment:
Transducer depth is simply pressure converted to depth. It is a simple derived variable, but I end up using it a lot when plotting and thus like to have it in the output dataset.

I agree that we should not assign standard names when there are none. I will see if there is something like "platform_depth" in the standards.

Reply:
Shouldn't the standard name just be `depth` then? If the long name is `transducer depth` and the variable is `xducer_depth` that seems clear to me... I suppose we could add an additional attribute called `info` that explains in more detail?

Reply:
I would just worry that `depth` collides with the coordinate depth. It's funny, CF has standard names for things like 'platform_heave', but not for the depth at rest. Maybe one would need to define a standard depth and then show the deviation from this as heave, but that defeats the purpose of having the ADCP depth quickly accessible.

Reply:
My understanding is that you can have a bunch of variables in a netCDF file with the same `standard_name`, and you differentiate between them in the `long_name`.  xarray always uses the long name first when labeling axes. 

I completely agree that it is funny to have so many platform/instrument specific variables (particularly heave!) but not depth. Maybe because of all the various ways of defining depth would just lead to a proliferation names  (Just search 'sea floor depth'... apparently this can be defined in many ways)?

Reply:
Okay, I agree and will set the standard name to `depth` here.